website/static/js/quickProjectSearchPlugin.js

```diff
@@ -0,0 +1,510 @@
+/**
+ * UI and function to quick search projects
+ */
+
+var $ = require('jquery');
+var m = require('mithril');
+var $osf = require('js/osfHelpers');
+
+// CSS
+require('css/quick-project-search-plugin.css');
+
+// XHR config for apiserver connection
+var xhrconfig = function(xhr) {
+    xhr.withCredentials = true;
+};
+
+
+var quickSearchProject = {
+    controller: function() {
+        var self = this;
+        self.nodes = m.prop([]); // Master node list
+        self.eligibleNodes = m.prop([]); // Array of indices corresponding to self.nodes() that are eligible to be loaded
+        self.sortState = m.prop('dateDesc'); //How nodes are sorted - default is date descending - dateDesc
+        self.countDisplayed = m.prop(); // Max number of nodes that can be rendered.  'Load more' increases this by up to ten.
+        self.next = m.prop(); // URL for getting the next ten user nodes. When null, all nodes are loaded.
+        self.loadingComplete = m.prop(false); // True when all user nodes are loaded.
+        self.contributorMapping = {}; // Maps node id to list of contributors for searching
+        self.filter = m.prop(); // Search query from user
+        self.fieldSort = m.prop(); // For xs screen, either alpha or date
+        self.directionSort = m.prop(); // For xs screen, either Asc or Desc
+
+        // Load first ten nodes
```

Comment:
The ten count comes from the default page size in the API, ya? I don't see that we're specifically requesting 10 nodes here ...